README.md

```diff
@@ -63,6 +63,18 @@ Here is an example of the JSON configuration file:
     }
   },
   "rebroadcastList": [
+    {
+      "name": "linkedin via restream",
```

Comment:
detonation names cannot have white space in them. See [README](https://github.com/michaelkamprath/multi-service-rtmp-broadcaster/blob/master/README.md).

Reply:
See #41